[rospy.client][INFO] 2021-10-22 11:35:20,312: [32minit_node, name[/ss42/inverse_kinematics_node], pid[4453][0m
[xmlrpc][INFO] 2021-10-22 11:35:20,314: [32mXML-RPC server binding to 0.0.0.0:0[0m
[xmlrpc][INFO] 2021-10-22 11:35:20,315: [32mStarted XML-RPC server [http://ss42.local:44823/][0m
[rospy.init][INFO] 2021-10-22 11:35:20,316: [32mROS Slave URI: [http://ss42.local:44823/][0m
[rospy.impl.masterslave][INFO] 2021-10-22 11:35:20,317: [32m_ready: http://ss42.local:44823/[0m
[rospy.registration][INFO] 2021-10-22 11:35:20,320: [32mRegistering with master node http://ss42.local:11311/[0m
[xmlrpc][INFO] 2021-10-22 11:35:20,321: [32mxml rpc node: starting XML-RPC server[0m
[rospy.init][INFO] 2021-10-22 11:35:20,417: [32mregistered with master[0m
[rospy.rosout][INFO] 2021-10-22 11:35:20,418: [32minitializing /rosout core topic[0m
[rospy.rosout][INFO] 2021-10-22 11:35:20,427: [32mconnected to core topic /rosout[0m
[rospy.simtime][INFO] 2021-10-22 11:35:20,432: [32m/use_sim_time is not set, will not subscribe to simulated time [/clock] topic[0m
[rosout][WARNING] 2021-10-22 11:35:20,445: [33m[/ss42/inverse_kinematics_node] /home/robotikabot/duckiefleet/calibrations/kinematics/ss42.yaml does not exist. Using default.yaml.[0m
[rosout][INFO] 2021-10-22 11:35:20,496: [32m[/ss42/inverse_kinematics_node] ~gain = 1.0 [0m
[rosout][INFO] 2021-10-22 11:35:20,507: [32m[/ss42/inverse_kinematics_node] ~trim = 0.0 [0m
[rosout][INFO] 2021-10-22 11:35:20,518: [32m[/ss42/inverse_kinematics_node] ~baseline = 0.1 [0m
[rosout][INFO] 2021-10-22 11:35:20,531: [32m[/ss42/inverse_kinematics_node] ~radius = 0.0318 [0m
[rosout][INFO] 2021-10-22 11:35:20,544: [32m[/ss42/inverse_kinematics_node] ~k = 27.0 [0m
[rosout][INFO] 2021-10-22 11:35:20,556: [32m[/ss42/inverse_kinematics_node] ~limit = 1.0 [0m
[rosout][INFO] 2021-10-22 11:35:20,635: [32m[/ss42/inverse_kinematics_node] Initialized.[0m
[rosout][INFO] 2021-10-22 11:35:20,637: [32m[/ss42/inverse_kinematics_node] gain: 1.0 trim: 0.0 baseline: 0.1 radius: 0.0318 k: 27.0 limit: 1.0[0m
[rospy.internal][INFO] 2021-10-22 11:35:20,646: [32mtopic[/ss42/joy_mapper_node/car_cmd] adding connection to [http://ss42.local:34328/], count 0[0m
[rospy.internal][INFO] 2021-10-22 11:35:20,674: [32mtopic[/rosout] adding connection to [/rosout], count 0[0m
[rospy.internal][INFO] 2021-10-22 11:35:20,694: [32mtopic[/ss42/wheels_driver_node/wheels_cmd] adding connection to [/ss42/wheels_driver_node], count 0[0m
[rospy.core][INFO] 2021-10-22 11:36:56,189: [32msignal_shutdown [signal-2][0m
[rospy.internal][ERROR] 2021-10-22 11:36:56,312: [31mUnable to initiate TCP/IP socket to ss42.local:36427 (http://ss42.local:34328/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down
[0m
[rospy.internal][INFO] 2021-10-22 11:36:56,340: [32mtopic[/ss42/wheels_driver_node/wheels_cmd] removing connection to /ss42/wheels_driver_node[0m
[rospy.internal][INFO] 2021-10-22 11:36:56,342: [32mtopic[/rosout] removing connection to /rosout[0m
[rospy.internal][INFO] 2021-10-22 11:36:56,343: [32mtopic[/ss42/joy_mapper_node/car_cmd] removing connection to http://ss42.local:34328/[0m
[rospy.impl.masterslave][INFO] 2021-10-22 11:36:56,345: [32msignal-2[0m
[rospy.core][INFO] 2021-10-22 11:36:56,811: [32msignal_shutdown [atexit][0m
